frontend/views.py

```diff
@@ -5344,10 +5344,11 @@ def clean_all_successful(request):
     to_update = []
     calcs = CalculationOrder.objects.filter(author=request.user, hidden=False)
     for c in calcs:
-        if c.status == 2:
-            c.hidden = True
-            c.see()
-            to_update.append(c)
+        if c.last_seen_status != 0:    #default value of c.last_seen_status is 0, if calculation is not viewed
```

Comment:
It should work if this condition is `if not c.new_status:` instead. This property is used to trigger the yellow outline in unseen calculations.

You can update the tests in `test_selenium.py` which verify that the cleaning functions are working properly (for example `test_clean_all_successful`). If the unseen calculation orders don't get cleaned, then those tests should fail because the outcome will be different. You should modify them so that they test properly that only unseen orders get cleaned.

Reply:
Okay, I will do it that way.

Reply:
Hello!
Shouldn't it be, if the unseen calculation orders get cleaned, then the tests should fail.
Because we want to clean only the seen calculation orders, leaving the unseen ones untouched.

Reply:
Yes, you are correct. I meant that some tests as they are right now could fail with your modification because they expect unseen calculation orders to get cleaned. You should update them so that they expect unseen calculation orders *not* to get cleaned, like you said.

Reply:
Okay, thanks for the clarification!